.evergreen/config.yml

```diff
@@ -264,16 +277,7 @@ functions:
         shell: "bash"
         script: |
           ${PREPARE_SHELL}
-          cd "$MONGO_ORCHESTRATION_HOME"
-          # source the mongo-orchestration virtualenv if it exists
-          if [ -f venv/bin/activate ]; then
-            . venv/bin/activate
-          elif [ -f venv/Scripts/activate ]; then
-            . venv/Scripts/activate
-          fi
-          mongo-orchestration stop
-          cd -
-          rm -rf $DRIVERS_TOOLS || true
+          bash ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh
```

Comment:
The tools script for stop-orchestration.sh doesn't include the logic that removes the drivers tools clone:

```bash
cd -
rm -rf $DRIVERS_TOOLS || true
```

Is this worth retaining as part of the cleanup?

Reply:
Re-added